Sentry.Xamarin.Forms/SentryXamarinFormsIntegration.cs

```diff
@@ -71,32 +114,11 @@ public void Register(IHub hub, SentryOptions options)
             {
                 Log.Listeners.Add(XamarinLogger);
             }
+        }
 
-            if (Options.Value.NativeIntegrationEnabled)
-            {
-#if NATIVE_PROCESSOR
-                Nativeintegration = new NativeIntegration(Options.Value);
-                Nativeintegration.Register(hub, options);
-#else
-                options.DiagnosticLogger.Log(SentryLevel.Debug, "No NativeIntegration found for the given target.");
-#endif
-            }
-
-            //Don't lock the main Thread while you wait for the current application to be created.
-            Task.Run(async () =>
-                {
-                    var application = await GetCurrentApplication().ConfigureAwait(false);
-                    if (application is null)
-                    {
-                        options.DiagnosticLogger.Log(SentryLevel.Warning, "Sentry.Xamarin.Forms timeout for tracking Application.Current. Navigation tracking is going to be disabled");
-                    }
-                    else
-                    {
-                        application.PageAppearing += Current_PageAppearing;
-                        application.PageDisappearing += Current_PageDisappearing;
-                        application.RequestedThemeChanged += Current_RequestedThemeChanged;
-                    }
-                });
+        internal void ConfigureSentryOptions(SentryOptions options)
+        {
+            options.Release ??= $"{AppInfo.PackageName}@{AppInfo.VersionString}";
```

Comment:
Don't you need to check if the user didn't explicitly define something?
Sounds like you'd be overriding their value here.

Reply:
Yes that's why I'm using ??=